ern-api-impl-gen/src/generators/android/ApiImplMavenGenerator.js

```diff
@@ -65,12 +65,12 @@ export default class ApiImplMavenGenerator implements ApiImplGeneratable {
       shell.cp(`-R`, path.join(paths.apiImplHull, 'android', '*'), outputDirectory)
 
       for (let plugin: Dependency of plugins) {
-        log.debug(`Copying ${plugin.name} to ${outputDirectory}`)
+        log.debug(`\nCopying ${plugin.name} to ${outputDirectory}`)
```

Comment:
Reason for `\n` here ? I guess there is one because you added it.
Looks odd. All log lines have a line return injected at the end by default, so I don't really see a reason.

Reply:
This was placed for debugging, forgot to remove.!